### ISSUE TITLE
"The "--database" option does not exist" error when attempting to install

### ISSUE THREAD

**Issue by philam** [ushahidi/platform#4515]
**Describe the bug**
I am trying to install [Ushahidi 5.3.1](https://github.com/ushahidi/platform) onto a shared web server.

I am following these official installation instructions here: https://docs.ushahidi.com/platform-developer-documentation/development-and-code/setup_alternatives/platform_release_install

I have extracted the files successfully as described and created a blank MySQL database with the appropriate permissions for the user.

When I populate the .env file and attempt to run `php artisan migrate` in a terminal in the appropriate folder (./platform), I get this error:

Database [<databasename>] not configured.

**To Reproduce**
1. Upload platform-release files to server.
2. Populate .env file.
3. Open terminal and navigate to /platform folder.
4. Attempt to run `php artisan migrate`
5. Error `The "--database" option does not exist` is displayed.

**Expected behaviour**
That `php artisan migrate` runs successfully and the installation can continue.

**Hardware Details:**
 - Apache 2.4.54
 - MySQL 10.3.37-MariaDB
 - Architecture x86_64
 - Linux
 - Perl version 5.16.3
 - PHP 7.4

**Additional context**
This is what the .env file looks like, with the appropriate private information snipped:

```
DB_HOST=localhost
DB_DATABASE=<databasename>
DB_USERNAME=<databasename>
DB_PASSWORD=<databasepassword>

APP_ENV=local
APP_DEBUG=false
APP_KEY=<32characterkey>
APP_TIMEZONE=UTC

CACHE_DRIVER=array
QUEUE_DRIVER=sync
```

I dug into the laravel.log file and this is the error message that comes up: https://pastebin.com/V0Qc8CZY

 - All modules listed in the installation docs confirmed as enabled
 - All privileges granted for the database user to the database

**What I have tried**

- Recreating the username and database in phpMyAdmin
- Trying different configurations of the .env file (e.g. changing the API key)
- Clearing the cache of php artisan
- Creating a new .env file

**Comment by tuxpiper** [comment]
That's quite unexpected as it seems to go into running Laravel migrations, whereas the project is set up to use phinx.

I'm not sure if it helps, but PHP 7.4 is not something the project has certified as running... maybe that's the root of the problem? PHP 7.3 is better tested.

Have you tried running the phinx commands explicitly?

```
php artisan phinx:migrate
```

**Comment by philam** [comment]
Thanks so much for the quick reply!

I just tried downgrading to PHP 7.3 but unfortunately with the same result.

When I tried running `php artisan phinx:migrate` I got the following error message (in both PHP 7.4 and 7.3):

```
There are no commands defined in the "phinx" namespace
```

To me this suggests that the module is not installed. I've contacted my web hosting provider for them to do so and I'll report back on what happens next.
